docs/advanced/bootloader.md

```diff
@@ -2,14 +2,18 @@
 
 This guide explains how to create a bootloader, how to create a main program to go with the bootloader and how to use this bootloader to perform software updates.
 
+## Porting bootloader
+
+Please follow [the bootloader porting guide](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/flash/) for adding a bootloader support for your target.
```

Comment:
OK will rephrase to:

A target requires must implement flash HAL functionality in order to support the bootloader. Please follow [the bootloader porting guide](https://docs.mbed.com/docs/mbed-os-handbook/en/latest/advanced/flash/) for more details.